explorer/templates/explorer/query_favorite_button.html

```diff
@@ -0,0 +1,8 @@
+{% if is_favorite %}
+    <i class="glyphicon glyphicon-heart text-danger query_favorite_toggle" data-id="{{ query_id }}"
+       style="cursor: pointer;"></i>
```

Comment:
I'd like to avoid inline styles.

Could you add the CSS to the end of `explorer.css` please?

Something like;
```
.query_favorite_toggle {
    cursor: pointer;
}

.query_favourite_detail {
    float: right;
}
```

And I'm curious what you think to moving the icon to the right. I'm not a huge fan of how it sits next to the title.

<img width="798" alt="Screenshot 2023-01-20 at 00 13 49" src="https://user-images.githubusercontent.com/1461191/213590158-d90daa92-7c8a-47ec-ad87-25a06e8515e2.png">